specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-04-01-preview/examples/KustoDatabasesListByKustoPool.json

```diff
@@ -0,0 +1,39 @@
+{
+  "parameters": {
+    "subscriptionId": "12345678-1234-1234-1234-123456789098",
+    "resourceGroupName": "kustorptest",
+    "workspaceName": "synapseWorkspaceName",
+    "kustoPoolName": "kustoclusterrptest4",
+    "api-version": "2021-04-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/12345678-1234-1234-1234-123456789098/resourceGroups/kustorptest/providers/Microsoft.Synapse/workspaces/kustorptest/kustopools/KustoClusterRPTest4/Databases/KustoDatabase8",
+            "name": "KustoClusterRPTest4/KustoDatabase8",
```

Comment:
>"name": "KustoClusterRPTest4/KustoDatabase8", [](start = 12, length = 45)

Why does the resource name had Kusto pool name too?

Reply:
it is sub-resource of kustopool